lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -255,12 +255,14 @@ sub filter_loh {
 #Get rid of intermediate files
 sub cleanup {
     my $self = shift;
-    my $unwanted = "*.Somatic* *.readcounts *.hc *.LOH" .
-        " *.lc *.Germline *.removed *.formatted" .
-        " *.hc.err *.other";
-    for my $file (glob $unwanted) {
-        $self->status_message("unlinking $file");
-        unlink $file;
+    my @patterns = qw(*.Somatic* *.readcounts *.hc *.LOH
+        *.lc *.Germline *.removed *.formatted
+        *.hc.err *.other);
+    for my $pattern (@patterns) {
+        for my $file (glob File::Spec->join($self->outdir, $pattern)) {
```

Comment:
Super nitpicky and maybe not worth implementing, but a `map` would eliminate the nested loops here.

````perl
my @files = map { glob File::Spec->join($self->outdir, $_) } qw(*.Somatic* *.readcounts *.hc *.LOH *.lc *.Germline *.removed *.formatted *.hc.err *.other);
$self->status_message("unlinking %s", join(',' @files));
unlink @files
````
You'd lose the ability to tell which ones failed or succeeded, but we aren't currently checking that anyways.